### PR TITLE
Add example feature

### PR DESCRIPTION
**Pull request by isaacwellish**
- Was able to add a feature to switch between a word's definition and an example sentence. 

- Only error I can't remove from pylint is:

`Access to a protected member _json_path of a client class (protected-access)`

### CODE REVIEW

File: PyPortal_Word_of_the_Day/code.py [hunk 1]
```diff
@@ -53,9 +56,18 @@
...
-    try:
-        value = pyportal.fetch()
-        print("Response is", value)
-    except RuntimeError as e:
-        print("Some error occured, retrying! -", e)
-    time.sleep(10*60) # Update every 10 minutes
+    if pyportal.touchscreen.touch_point:
+        try:
+            #set the JSON path here to be able to change between definition and example
+            pyportal._json_path=(WORD_LOCATION,
```
Comment (brentru) [inline PyPortal_Word_of_the_Day/code.py]: I have no issues with accessing this path as it's not an exposed property and there's no other way of changing it.

* Before Line 62, add the following line: 
`# pylint: disable=protected-access`

File: PyPortal_Word_of_the_Day/code.py [hunk 2]
```diff
@@ -20,7 +20,11 @@
 WORD_LOCATION = ['word']
```
Comment (brentru) [inline PyPortal_Word_of_the_Day/code.py]: This doesn't need to be predefined, just use `['word']` on Line 62.

File: PyPortal_Word_of_the_Day/code.py (hunk 1 above)
Comment (isaacwellish) [reply PyPortal_Word_of_the_Day/code.py]: Got it!

File: PyPortal_Word_of_the_Day/code.py (hunk 2 above)
Comment (isaacwellish) [reply PyPortal_Word_of_the_Day/code.py]: kk!

**Comment by brentru** [comment]
@isaacwellish  tag me again when you're ready to re-review and merge

**Comment by isaacwellish** [comment]
@brentru ready to go!

**Comment by brentru** [comment]
LGTM, thanks!

**Pull request merged**